front/index.html

```diff
@@ -1,12 +1,12 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="h-full bg-white">
```

Comment:
You should remove that

Reply:
I used this class to give a height of 100% and to give a white background to all the html element.